src/content/upgrades/merge/index.md

```diff
@@ -5,64 +5,236 @@ lang: en
+
+[More on Ethereum security](/security/#eth2-token-scam)
+
+### Node operators and dapp developers {#node-operators-dapp-developers}
+
+<ExpandableCard
+title="Staking node operators and providers"
+contentPreview="If you are a staker running your own node setup or a node infrastructure provider, there are a few things you need to be aware of to be prepared for The Merge."
+id="staking-node-operators">
+Key action items include:
+
+- Running _both_ a consensus layer client as well as an execution layer client. Third-party endpoints to obtain execution data will be unavailable after The Merge.
+- Authenticating both execution layer and consensus layer clients with a shared JWT secret so they can securely communicate with one another.
+
+Not completing the above items will result in your node being seen as "offline" after The Merge until both layers are synced and authenticated.
+
+- Setting a `fee recipient` address to direct your earned transaction fee tips/MEV to.
+
+Not setting a `fee recipient` will still allow your validator to behave as usual, but you will miss out on unburnt fee tips and any MEV you would have otherwise earned in blocks your validator proposes.
+
+For more detailed information and a summary of links to client resources, stakers are encouraged to check out the [Merge Readiness Checklist](https://launchpad.ethereum.org/en/merge-readiness/) over on the Staking Launchpad to make sure you're fully prepared for The Merge.
+
+Note for stakers using [SaaS](/staking/saas/) or [staking pools](/staking/pools/): There is nothing you need to do to prepare for The Merge. [More below on staying safe.](#users-holders)
+</ExpandableCard>
+
+<ExpandableCard
+title="Non-validating node operators and infrastructure providers"
+contentPreview="If you're operating a non-validating Ethereum node, the most significant change that comes with The Merge is the requirement to run clients for BOTH the execution layer AND the consensus layer."
+id="node-operators">
+You probably are already running an execution layer client, such as Geth, Erigon, Besu or Nethermind. Up until The Merge, an execution layer client was enough to receive, properly validate, and propagate blocks being gossiped by the network. _After The Merge_, the validity of transactions contained within an execution payload will also depend on the validity of the "consensus block" it is contained within.
+
+This means that a full Ethereum node after The Merge requires both an execution layer client as well as a consensus layer client. These two clients work tightly together using a new Engine API to properly determine the latest state of the network. The Engine API requires authentication using a JWT secret, which is provided to both clients allowing secure communication.
```

Comment:
unclear what 'this' refers to

```suggestion
As a result, a full Ethereum node after The Merge requires both an execution layer client and a consensus layer client. These two clients work together using a new Engine API to correctly determine the latest state of the network. The Engine API requires authentication using a JWT secret, which is provided to both clients allowing secure communication.
```

Reply:
"...a full Ethereum node after The Merge **comprises** both an execution..."

do we have to use "execution *layer* client"? 

The engine API is not really about state management - this is still in the remit of the execution client. The engine-API is about enabling the consensus client to make requests to the execution client and for the execution client to return execution payloads and other information back up to the consensus client.

Just kill the sentence at "...using a new Engine API"?

Reply:
@minimalsm "As a result" of what? 😆